content/derisking/federal-field-guide/4-deciding-what-to-buy.md

```diff
@@ -0,0 +1,698 @@
+
+The traditional government fear of T&M contracts is that costs spin out of control. The empowered product owner[^owner] reduces this risk though daily communication, and frequent — every sprint — inspection of performance against a quality assurance surveillance plan (QASP). We also encourage using a not-to-exceed ceiling to put an additional cap on expenses to protect the government's interest.
+
+In addition to flexibility, another T&M benefit is that it allows for development team flexibility to scale up and down as needed. Stable teams are a core tenet of agile development. agile frowns on changing team structures more often than needed, but T&M allows the product owner to do so if necessary.
+
+A real-life example: a new government product owner rightly determined that they could only manage one scrum team since the role was new. Six months into the project, the government product owner had mastered the role and was confident in their abilities. They decided to add an additional scrum team to speed development work. Once the key functionality has been built, the government product owner plans to return to one scrum team, likely smaller than the original. These changes are made possible by using T&M. No modifications are necessary because the agency is paying for the time humans spend building out the product — the number of humans and which roles are filled can change.
+
+T&M allows agencies to save money, too. The contractor can only bill for time spent building a product. If there are delays hiring a new team member or a person goes on extended leave, the government is not paying for that time. T&M provides the opportunity for costs to come in under budget, which never happens with FFP.
+
+A final T&M benefit is that if the contractor is not producing quality software — if their work is inadequate or their skills prove to not fit the work needed — no more work needs be assigned to the contractor, which effectively terminates the contract. There is no work in the product backlog, and the contractor cannot bill their time to the government. There is no need to terminate for convenience or cause.
+
+To be clear, T&M does require more active management than the FFP "set it and forget it" contracts. The government must know what is going on, and review and approve invoices appropriately. Luckily, with agile development, this should already be happening. Read more about this shift in our recommendation on [post-award management](/federal-field-guide/doing-the-work/#post-award-contract-administration-looks-different-in-agile).
```

Comment:
This is the broken link that probably needs a `| url`

Reply:
Must have been afternoon brain — I could _not_ find this yesterday. Thank you. Fixed.